### ISSUE TITLE
Browser back button does not update the router by default

### ISSUE THREAD

**Issue by jdin** [Gladear/haunted-router#1]
Hi @Gladear 
First of all, thanks for the library!

But I have a question.
I use links for navigation without `replace` to allow user to go back if needed.
But I have noticed that by default it does not work and I haven't found any listener for `window.onpopstate` in your code. So I had to add the following to my project:
```javascript
import {update} from 'haunted-router/lib/router.js';
window.onpopstate = update;
```
Was it the meaning to do it this way or do I miss something?

**Comment by Gladear** [comment]
Definitely something I missed !
Thanks for reporting the issue, i'll correct it asap 😊

**Comment by Gladear** [comment]
Fixed with v0.1.3
